src/PluginVm.ts

```diff
@@ -416,6 +416,7 @@ export class PluginVm {
 			// override env to "unlink" from original process
 			const srcEnv = sandboxTemplate.env || global.process.env;
 			sandbox.process.env = {...srcEnv}; // copy properties
+			(sandbox as any).process.on = (event: string, callback: any) => {};
```

Comment:
I'm guessing it wont be very secure to map `process.on` to node's `process.on` events?

Reply:
Yes, agree!